### PR TITLE
Enforce naming convention

### LINKED ISSUE #1769: Adopt and enforce naming convention in ts-tests code
### Context

Mixed naming styles (e.g. `snake_case` for some variables and `camelCase` for others) add unnecessary friction when reading through the test code.

### Task

- Enable [this linting rule on the TS code](https://typescript-eslint.io/rules/naming-convention/)
- Configure the rule once consensus is achieved on the convention to adopt

### Acceptance criteria

- Lint checks pass with this rule enabled

### Questions/Blockers

- [ ] Discussion still ongoing on which convention to adopt

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

resolves #1769

Mainly updates:

- Fixed improper usage of var, let, and const.

- Corrected improper naming conventions.

### Testing Evidences

image

Please disregard this warning. It is a leftover `TODO` item that is not intended to be addressed in this PR.

### CODE REVIEW

**Comment by 0xverin** [comment]
Please hold off on the review for now. I would like to check the CI status first.

**Comment by Verin1005** [comment]
Once CI passes, it can be reviewed.

**Comment by 0xverin** [comment]
> Please hold off on the review for now. I would like to check the CI status first.

Once CI passes, it can be reviewed.

**Comment by 0xverin** [comment]
Do we need to perform ESLint checks when submitting code? I think it may not be necessary.

- VSCode checks naming conventions based on our `.eslintrc.json` file, so we can pay attention to it regularly.

- The `.git` files are not in the `ts-tests` directory. If we use `pre-commit` hooks, we would need to install ESLint plugin separately in the `/parachain` directory, which is not appropriate. We don't need to execute `yarn` to install dependencies in the `/parachain` directory.

File: tee-worker/ts-tests/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"parser": "@typescript-eslint/parser",
+	"plugins": ["@typescript-eslint"],
+	"root": true,
+	"rules": {
+		/**
+    It's a temporary solution, folks. We had no choice but to shut it off,
+    because there's just a liiittle bit too much "any" lurking around in the code.
+    But fear not, my friends, for this is not the end of the story.
+    We shall return, armed with determination and resolve,
+    to tackle those "any" types head-on in the near future.
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/.eslintrc.json:12]: <3

**Review by arieljbirnbaum — APPROVED** [review]
Awesomesauce! Thank you :raised_hands:

**Pull request merged**